models/quality_measures/intermediate/nqf0053_osteoporosis_management_in_women_who_had_a_fracture/quality_measures__int_nqf0053_denominator.sql

```diff
@@ -0,0 +1,297 @@
+        , patients_with_age.max_age as age
+        , pp.performance_period_begin
+        , pp.performance_period_end
+        , pp.measure_id
+        , pp.measure_name
+        , pp.measure_version
+        , 1 as denominator_flag
+    from bone_fracture_conditions
+    left join patients_with_age
+        on bone_fracture_conditions.patient_id = patients_with_age.patient_id
+    cross join {{ref('quality_measures__int_nqf0053__performance_period')}} pp
+    where max_age >= 50 and min_age <=  85
+        and bone_fracture_conditions.recorded_date between
+            {{ dbt.dateadd (
+              datepart = "month"
+            , interval = -6
+            , from_date_or_timestamp = "performance_period_begin"
+            )
+            }}
+            and
+                pp.lookback_period_june
+        and lower(patients_with_age.sex) = 'female'
+
+)
+
+, fracture_procedures as (
+
+    select
+        procedures.*
+    from procedures
+    inner join visit_codes
+        on procedures.source_code = visit_codes.code
+            and procedures.source_code_type = visit_codes.code_system
+    inner join {{ ref('quality_measures__int_nqf0053__performance_period') }} as pp
+        on procedures.procedure_date 
+            between pp.performance_period_begin and pp.performance_period_end
+    where lower(visit_codes.concept_name) = 'fracture procedures'
+
+)
+
+, qualifying_patients_w_encounter as (
+
+    select
+        qualifying_patients_w_fractures.*
+    from qualifying_patients_w_fractures
+    inner join visits_encounters
+        on qualifying_patients_w_fractures.patient_id = visits_encounters.patient_id
+    where 
+        lower(visits_encounters.encounter_type) in (
+              'acute inpatient'
+            , 'annual wellness visit'
+            , 'emergency department visit'
+            , 'emergency department'
+            , 'home healthcare services'
+            , 'office visit'
+            , 'preventive care services established office visit, 18 and up'
+            , 'preventive care services initial office visit, 18 and up'
+            , 'emergency department evaluation and management visit'
+            , 'outpatient'
+        )
+)
+
+, qualifying_patients_w_procedure as (
+
+    select
+        qualifying_patients_w_fractures.*
+    from qualifying_patients_w_fractures
+    inner join fracture_procedures
+        on qualifying_patients_w_fractures.patient_id = fracture_procedures.patient_id
+
+)
+
+, qualifying_patients as (
+
+    select
+        distinct
+        qualifying_patients_w_encounter.*
+    from qualifying_patients_w_encounter
+    left join qualifying_patients_w_procedure
+        on qualifying_patients_w_encounter.patient_id = qualifying_patients_w_procedure.patient_id
+    inner join claims_encounters
+        on qualifying_patients_w_encounter.patient_id = claims_encounters.patient_id
+    where cast(claims_encounters.place_of_service_code as {{ dbt.type_string() }}) not in ('21')
```

Comment:
I've been testing on our customer data. This join is causing issues with customers that do not have claims and only have clinical sources. We should adjust the logic (see below) to use the place of service when claims_encounters are present.

```suggestion
    left join claims_encounters
        on qualifying_patients_w_encounter.patient_id = claims_encounters.patient_id
    where (cast(claims_encounters.place_of_service_code as {{ dbt.type_string() }}) not in ('21')
        or claims_encounters.patient_id is null)
```